kratos/processes/apply_periodic_boundary_condition_process.cpp

```diff
@@ -149,38 +149,30 @@ template <int TDim>
 void ApplyPeriodicConditionProcess::ApplyConstraintsForPeriodicConditions()
 {
     const auto timer = BuiltinTimer();
-    const int num_vars = mParameters["variable_names"].size();
+    const unsigned int num_vars = mParameters["variable_names"].size();
     BinBasedFastPointLocatorConditions<TDim> bin_based_point_locator(mrMasterModelPart);
     bin_based_point_locator.UpdateSearchDatabase();
 
-    IndexType num_slaves_found = 0;
-
-    num_slaves_found = block_for_each<SumReduction<IndexType>>(mrSlaveModelPart.Nodes(), [&](Node<3>& rNode){
-        IndexType counter = 0;
-        Condition::Pointer p_host_cond;
-        VectorType shape_function_values;
-        array_1d<double, 3 > transformed_slave_coordinates;
-        TransformNode(rNode.Coordinates(), transformed_slave_coordinates);
+    struct auxiliary {IndexType counter = 0; Condition::Pointer p_host_cond; VectorType shape_function_values; array_1d<double, 3 > transformed_slave_coordinates;};
```

Comment:
```suggestion
    struct auxiliar {IndexType counter = 0; Condition::Pointer p_host_cond; VectorType shape_function_values; array_1d<double, 3 > transformed_slave_coordinates;};
```

Reply:
Jokes aside, I'd call this `TLS` or `TLS_container`.